app/javascript/react/components/ReviewTile.js

```diff
@@ -1,15 +1,38 @@
-import React from 'react';
-
-const ReviewTile = (props) => {
-  return(
-    <div>
-      {props.rating} <br />
-      {props.body}<br />
-      {props.user}<br />
-      {props.created_at}<br /> <br /> <br />
-      <a href={`/reviews/${props.id}/edit`}> Edit Review </a>
-    </div>
-  )
+import React, { Component } from 'react';
+
+class ReviewTile extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+
+    }
+    this.handleDelete = this.handleDelete.bind(this)
+  }
+    handleDelete() {
+      fetch(`/api/v1/reviews/${this.props.id}`, {
+        credentials: 'same-origin',
+        method: 'DELETE',
+        headers: { 'Content-Type': 'application/json'}
+       })
+       .then(response => response.json())
+       .then( response => {
+         return response;
+       })
+       .then(location.reload(true))
```

Comment:
wat. ༼ʘ̚ل͜ʘ̚༽

Are you reloading the window or the component after a deletion? Regardless this is not ideal. Your handle delete method should be bound higher where your reviews state array is defined, and then passed down so that ReviewTiles have onClick triggers for it